<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TapeSpltClass</class>
 <widget class="QDialog" name="TapeSpltClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>621</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Input File</string>
   </property>
   <widget class="QLineEdit" name="inputFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="inputSelButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>621</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Output File(s)</string>
   </property>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier 10 Pitch</family>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
